client/components/PetTile.js

```diff
@@ -1,11 +1,19 @@
-import React from 'react'
-
-const PetTile = () => {
-  return (
-    <div>
-      
-    </div>
-  )
-}
-
-export default PetTile
+import React from 'react'
+import { Link } from "react-router-dom"
+
+const PetTile = (props) => {
+  const {id, name, description, imageUrl} = props.pet
+  
+  return (
+    <div>
+      <div>
```

Comment:
Shouldn't need this secondary `div`, especially if no styling or class names are being applied